### DISCUSSION TITLE
Migration from docsify

### DISCUSSION THREAD

**Discussion by Piotr1215** [facebook/docusaurus#5894] [Q&A] [answered]
EDIT: Added embedded, indented HTML support

I'm new to docosaurus and I do apologize if some of the questions are obvious, but I couldn't find the answers in the documentation.

I'm planning to migrate [my documentation/blog](https://www.itcertificationsguide.com/#/?id=introduction) from docsify to docosaurus. The 2 main reasons are:
- versioning support
- blogs engine combined with the docs (this is a killer feature for me, as my page is mix of blog-style content and documentation)

I'm not sure if there is an official migration guide, but even if not, I can quickly copy markdown over. There are however a few plugins I'm using and I couldn't find in the docs if the following functionality is supported:

- [x] remote markdown: ability to render markdown files from a remote location (another repo of gist/snippet)
- [x] router to external content: similar like below, but instead of remote rendering, it just internally routes. Not sure if this plugin supports this functionality (https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-client-redirects)
- [x] correctly parsing embedded, indented HTML

**Comment by juzhiyuan** [comment]
Ok to do so, the direction is putting your Markdown files to the new website 😄

**Comment by juzhiyuan** [comment]
BTW, it seems that your site is similar to Apache APISIX's, it includes blogs & docs (many projects) as well.

codes: https://github.com/apache/apisix-website
site: https://apisix.apache.org/

**Reply by Piotr1215** [reply]
This a gorgeous looking website :). I've experimented yesterday a bit the moving some of my marodown over, but noticed errors with embedded HTML. I'm using details tag to hide responses to questions on the pages, like so (they are also indented towards right):

Inside of those tags there are other embedded and it looks like docosaurus enforces HTML correctness  (which is good), but is there another way of doing it or that's a recommended way in dososaudus too?

```
    <details>
    <summary>click to see the answer</summary>

      Answer here

    </details>
```

How about remote markdown rendering, is it possible?

**Reply by Josh-Cena** [reply]
Not accepting indented HTML is not intended behavior, but a technical limitation of our underlying MDX parser. Once we migrate to MDX v2 (#4029) your usage will be supported.

**Reply by Piotr1215** [reply]
Great! I'll keep an eye on this issue. Updated the thread with remaining points for easier tracking.

**Reply by Piotr1215** [reply]
I was able to move my web page to defosaurus after the migration to MDX2 🎉

**Comment by Piotr1215** [comment, marked as answer]
I was able to move my web page to docosaurus after the migration to MDX2 :tada:
